app/src/main/java/com/release/keyneez/presentation/main/explore/recent/RecentFragment.kt

```diff
@@ -0,0 +1,40 @@
+package com.release.keyneez.presentation.main.explore.recent
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.viewModels
+import com.release.keyneez.R
+import com.release.keyneez.databinding.FragmentExploreRecentBinding
+import com.release.keyneez.domain.model.ExploreData
+import com.release.keyneez.util.binding.BindingFragment
+
+class RecentFragment :
+    BindingFragment<FragmentExploreRecentBinding>(R.layout.fragment_explore_recent) {
+
+    private val viewModel: RecentViewModel by viewModels()
+    val data = mutableListOf<ExploreData>()
+    private lateinit var RecentAdapter: RecentAdapter
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setupRecentData()
+        initRecentAdapter()
+    }
+
+    private fun setupRecentData() {
+        viewModel.itemList.observe(viewLifecycleOwner) {
+            RecentAdapter.data = it
+            RecentAdapter.notifyDataSetChanged()
+        }
```

Comment:
```suggestion
        viewModel.activityList.observe(viewLifecycleOwner) { activityList ->
            recentAdapter.data = activityList
            recentAdapter.notifyDataSetChanged()
        }
```
요기도 item이라는 명칭 보다는 activity를 사용하면 좋을 것 같구, `it` 보다는 어떤 변수인지 명시해주는 것이 보기 좋을 것 같습니다!
(ListAdapter와 관련된 리뷰는 일단 생략하겠습니다! 나중에 꼭 바꿔주셔야해요... 🥹)

Reply:
넵! 코드 이것저것 짜다보니 까먹고 익숙한 방식으로 짰네요ㅠㅠ